README.md

```diff
@@ -77,6 +77,17 @@ Start your interactive ruby shell `irb`:
 - Twitter: [@twitterhandle](https://twitter.com/HenaryKhuraijam)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/henary-khuraijam-50487317a/)
 
+👤 **Lucy W. Mwangi**
+
+- GitHub: [@lucy-sees](https://github.com/lucy-sees)
+- Twitter: [@lucy_w_mwangi](https://twitter.com/lucy_w_mwangi)
+- LinkedIn: [@Lucy (Wanjiru) Mwangi](https://www.linkedin.com/in/lucy-wanjiru-mwangi)
+
+👤 **AishaAlsewi**
+
+- GitHub: [Aisha Alsewi](https://github.com/AlselwiAisha)
+- LinkedIn: [Aisha Alsewi](https://www.linkedin.com/in/aisha-alselwi/)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] The issues page link in the README file is wrong, It's a link to the repository. Please make sure to fix this, it will make the README file more professional.